cmd/validate-krew-manifest/main.go

```diff
@@ -39,9 +39,11 @@ import (
 )
 
 var flManifest string
+var flInstall bool
 
 func init() {
 	flag.StringVar(&flManifest, "manifest", "", "path to plugin manifest file")
+	flag.BoolVar(&flInstall, "install", true, "install the plugin as part of the validation")
```

Comment:
I think it could be better to have a `-skip-install` what do you think?

Reply:
sounds good!

Reply:
pushed @ahmetb